lib/services/elasticsearch.js

```diff
@@ -286,6 +359,12 @@ function ElasticSearch (kuzzle, options, config) {
     var
       data = cleanData.call(this, requestObject);
 
+    // Add metadata
+    data.body._kuzzle_info = {
+      updatedAt: new Date(),
```

Comment:
Same as above: 

```new Date()``` will return a date formatted with local time (ex: ```Mon Oct 17 2016 14:34:29 GMT+0000 (UTC)```) 

I'll suggest to use ```Date.now()``` function instead.
Or define mapping to recognize a specific date format and format date field with momentjs: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html 
http://momentjs.com/docs/#/displaying/format/